CMakeLists.txt

```diff
@@ -29,12 +39,24 @@ target_include_directories(appx
                            PrivateHeaders
                            ${OPENSSL_INCLUDE_DIR}
                            ${ZLIB_INCLUDE_DIRS})
-target_link_libraries(appx
+						   
+						   
+if(STATIC_BUILD)
+ target_link_libraries(appx
```

Comment:
Style nits: the rest of the file uses a two space indent and aligns continuation lines one column after the opening parenthesis.